shiny/workflowPlot/server.R

```diff
@@ -89,7 +94,33 @@ server <- shinyServer(function(input, output, session) {
     }
     return(globalDF)
   })
-  # Renders the ggplotly 
+  # Allows to load actual data (different from model output) following the tutorial
+  # https://github.com/PecanProject/pecan/blob/develop/documentation/tutorials/AnalyzeOutput/modelVSdata.Rmd
+  # @params: bety,settings,File_path,File_format
+  loadObservationData <- function(bety,settings,File_path,File_format){
+    start.year<-as.numeric(lubridate::year(settings$run$start.date))
+    end.year<-as.numeric(lubridate::year(settings$run$end.date))
+    site.id<-settings$run$site$id
+    site<-PEcAn.DB::query.site(site.id,bety$con)
+    observations<-PEcAn.benchmark::load_data(data.path = File_path, format= File_format, time.row = File_format$time.row,  site = site, start_year = start.year, end_year = end.year) 
+    return(observations)
+  }
+  getFileFormat <- function(bety,format.id){
```

Comment:
This function has been implemented as a wrapper to retrieve the file format either on input id or on format id. For now, it is just based on file format. In future if we allow the user to retrieve based on input id as well.